src/int/limb.rs

```diff
@@ -198,7 +199,8 @@ macro_rules! define {
 
         /// Subtract [`i32`] from the big integer.
         ///
-        #[doc = $crate::shared::docs::limb_doc!(subtraction)]
+        #[doc = $crate::shared::docs::fixed_doc!(addition)]
```

Comment:
This needs to be corrected back to subtraction, there's more instances in this file.